pkg/test/framework/README.md

```diff
@@ -113,7 +113,13 @@ Several flags are available to customize the behavior, such as:
 | istio.test.ci | `false` | Enable CI Mode. Additional logging and state dumping will be enabled. |
 | istio.test.kube.config | `~/.kube/config` | Location of the kube config file to be used. |
 | istio.test.kube.minikube | `false` | If `true` access to the ingress will be via nodeport. Should be set to `true` if running on Minikube. |
-| istio.test.kube.systemNamespace | `istio-system` | Namespace for Istio deployment. If '', the namespace is generated with the prefix "istio-system-". |
+| istio.test.kube.systemNamespace | `istio-system` | Depreciated, namespace for Istio deployment. If '', the namespace is generated with the prefix "istio-system-". |
```

Comment:
Typo: should be "deprecated" instead. Similar places below.

Reply:
Thanks, done.